skyxplore-ui-test/src/test/java/com/mentoring/pageobject/FactoryPage.java

```diff
@@ -25,13 +28,16 @@
     @FindBy(xpath = "//div[@class='queue-element-title']/span[contains(text(),'CEX-01 Csatlakozó bővítő')]")
     private WebElement cex01ProductionQueueItem;
 
+    @FindBy(css = ".queue-element")
+    private WebElement queueElement;
+
     @FindBy(xpath = "(//div[@id='queue']//span)[3]")
     private WebElement quantityInQueue;
 
     @FindBy(css = ".queue-process .process-bar-text")
     private WebElement queueProcessBar;
 
-    FactoryPage(WebDriver driver) {
+    public FactoryPage(WebDriver driver) {
```

Comment:
This can be kept package private if no extra instantiation is made on test level (see my comment there)

Reply:
modified.